docs/changes.md

```diff
@@ -1,6 +1,8 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+- `12.8` Multiple Updates:
+    - Extend support for BIRT report configuration in Manage's server bundle pods ([#601](https://github.com/ibm-mas/ansible-devops/pull/601))
```

Comment:
As with other PR, please replace with the slightly shorter PR title so that it's consistent